WordPress/src/androidTest/java/org/wordpress/android/e2e/BlockEditorTests.java

```diff
@@ -41,6 +40,8 @@ public void setUp() {
 
     @Test
     public void publishSimplePost() {
+        String mTitle = new Throwable().getStackTrace()[0].getMethodName();
```

Comment:
Having current test name to be used as a post header, for easier navigation in FTL video recording.

Reply:
Great thinking @pachlava and thank you for making this change to make this easier to debug! ❤️ 

1) Suggestion (💡): Just an FYI, and in case you find this information useful, that `m` prefix that you see here is only used for fields (not variables) and it is called [hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation). Now, this notation is mostly no longer utilized in Android development, as such engineers tend to avoid that or refactor it when they see it still existing (just like in this case). Thus, please consider renaming `mTitle` to plain `title`.
2) Suggestion (💡): Although this `new Throwable().getStackTrace()[0].getMethodName()` bit of code is smart and gives you the method name auto-magically, I would recommend keeping it simple by using `String title = "Publish Simple Post"`. Especially since this `...getMethodName()` piece of code is not being reused, thus you needing to type it for every test, using an explicit string for every test, might have the same effect, plus making thing simpler. PS: Same applies for `publishFullPost()` and `blockEditorCanDisplayElementAddedInHtmlMode()` tests. Wdyt? 🤔

Reply:
@ParaskP7, thanks for the details behind `m` prefix, I wasn't aware of Hungarian notation!

> Although this `new Throwable().getStackTrace()[0].getMethodName()` bit of code is smart and gives you the method name auto-magically, I would recommend keeping it simple by using String title = "Publish Simple Post". Especially since this ...getMethodName() piece of code is not being reused, thus you needing to type it for every test, using an explicit string for every test, might have the same effect, plus making thing simpler.

Because I try to avoid code duplication as much as I can, using a hardcoded strings did not seem like an option to me, because I thought - "hey, why should I repeat it, if I can get it this way?". However, in this case, I agree with you: what I did looks like an overkill just for the sake of it, and I don't even understand how exactly it works 😅 .

I addressed both `m` prefix issue and post titles values in 9252be48084b368750ddc4d88a6d17e080c3c0b7. However, I went for the full copy of method name as a value, and not as separate nice human-words, mostly for the case of potentially easier renaming in the future. Thanks!

Reply:
Thank you so much for applying this suggestion @pachlava , I reviewed it and it LGTM! 🙇 ✅

> I went for the full copy of method name as a value, and not as separate nice human-words, mostly for the case of potentially easier renaming in the future.

This totally makes sense, I like! 💯